src/hints/bootloader_hints.rs

```diff
@@ -744,7 +742,8 @@ mod tests {
         assert!(is_plain(&mut vm, &mut exec_scopes, plain_packed_output));
 
         // Increment AP to avoid an inconsistent memory error writing in the same slot
-        vm.run_context.ap += 1;
+        let new_ap = (*&vm.get_ap() + 1usize).unwrap();
```

Comment:
Why `*&`? Seems weird.

```suggestion
        let new_ap = vm.get_ap().offset + 1;
```

Reply:
Resolved by https://github.com/Moonsong-Labs/cairo-bootloader/pull/4/commits/81deb991e449e74e7823e73a6a99a0579093f617